init-tools.msbuild

```diff
@@ -12,5 +15,18 @@
     <PackageReference Include="Microsoft.DotNet.BuildTools.Coreclr">
       <Version>1.0.4-prerelease</Version>
     </PackageReference>
+    <PackageReference Include="$(FeedTasksPackage)">
+      <Version>$(FeedTasksVersion)</Version>
+    </PackageReference>
   </ItemGroup>
+
+  <Target Name="CopyFeedTasks" AfterTargets="Restore" >
+    <ItemGroup>  
+      <FilesToCopy Include="$(MSBuildThisFileDirectory)\packages\$(FeedTasksPackage)\$(FeedTasksVersion)\lib\**\*.*"/>  
+    </ItemGroup>  
+
+    <Copy SourceFiles="@(FilesToCopy)"
+          DestinationFiles="@(FilesToCopy->'$(MSBuildThisFileDirectory)\Tools\$(FeedTasksPackage)\%(RecursiveDir)%(Filename)%(Extension)')"
```

Comment:
We shouldn't make an assumption where the Tools directory is. We should pass that in. I say that because as part of the source-build work we will be overriding this path.